src/shared/services/commandInterpreterHelper.js

```diff
@@ -722,41 +724,59 @@ const availableCommands = [
             result: grassData
           })
         )
-      } else if (param === 'reset') {
-        put(updateGraphStyleData(null))
-      } else if (isValidURL(param)) {
-        if (!param.startsWith('http')) {
-          param = `http://${param}`
-        }
+      }
+
+      function updateGrassAndShow(newGrass) {
+        put(updateGraphStyleData(newGrass))
+        showGrass()
+      }
+
+      function putErrorFrame(error) {
+        put(
+          frames.add({
+            useDb: getUseDb(store.getState()),
+            ...action,
+            error,
+            type: 'error'
+          })
+        )
+      }
+
+      if (param === '') {
+        showGrass()
+        return
+      }
+
+      if (param === 'reset') {
+        updateGrassAndShow(null)
```

Comment:
I've changed it to show a new pane on `:style reset` and `:style node {color: red}` to make it more obvious that something actually happens. If a user wants to see how their graph is changed they'd need to pin it for it not to scroll away though. It's not perfect this way but we want to rework how grass-editing works so it good for now